compatibility_server/pip_checker.py

```diff
@@ -352,6 +401,8 @@ def _install(self, container: docker.models.containers.Container):
         if returncode:
             environment_error = PIP_ENVIRONMENT_ERROR_PATTERN.search(output)
             if environment_error:
+                MMAP.measure_int_put(DOCKER_ERROR_MEASURE, 1)
```

Comment:
This isn't necessarily a docker error. I think that you can remove this check.

Reply:
done